pylearn2/costs/ebm_estimation.py

```diff
@@ -35,11 +35,13 @@
 # precisely when you're going to exceed the stack segment.
 sys.setrecursionlimit(40000)
 
+
 class NCE(DefaultDataSpecsMixin, Cost):
     """
     Noise-Contrastive Estimation
 
-    See "Noise-Contrastive Estimation: A new estimation principle for unnormalized models "
+    See "Noise-Contrastive Estimation:
+         A new estimation principle for unnormalized models"
     by Gutmann and Hyvarinen
 
     Parameters
```

Comment:
I'm not sure what is wrong with your change, but it makes the script generating the documentation crash.
To manually run that script locally, so you can check that your fix works, you can run:
`python doc/scripts/docgen.py --test` from the `Pylearn2` directory, for instance.